src/connect.ts

```diff
@@ -1,18 +1,17 @@
-import mongoose from "mongoose"
+import mongoose from "mongoose";
 
-export default (db:string) =>{
-    const connect = () =>{
-        mongoose
-            .connect(db,{useNewUrlParser:true})
-            .then(()=>{
-                return console.log(`Succesfully connected to ${db}`)
-            })
-            .catch(error =>{
-                console.log("Error connecting to database : ", error)
-            });
-    }
-    connect()
-
-    mongoose.connection.on("disconnected", connect);
-}
+export default (db: string) => {
```

Comment:
Db parameter, what does this mean, are you passing the db as a parameter :))? Maybe dbConnectionString?